doc/sphinx/addendum/type-classes.rst

```diff
@@ -521,6 +559,20 @@ Settings
    the dependent ones previously (Coq 8.5 and below). This can result in
    quite different performance behaviors of proof search.
 
+   .. _TypeclassesDefaultMode:
+
+.. opt:: Typeclasses Default Mode {? ( {| "+" | "-" | "!" } ) }.
+
+   This option (unset by default) controls the default mode declaration
+   associated to a :cmd:`Class`. If set, the each class declaration uses
+   this default mode for *all* its indices, unless an :attr:`mode` attribute
+   is used to set the mode explicitly.
```

Comment:
`Set Typeclasses Default Mode.` fails asking for one of `+/-/!`, `Unset Typeclasses Default Mode` unsets it, so I think that's the right syntax? It is maybe not entirely self evident given the grammar that "Unset" can be used.

Reply:
Maybe I know about this too well, but I think the syntax is pretty clearly documented: https://coq.inria.fr/refman/language/core/basic.html#flags-options-and-tables

Reply:
Ah, but Jim's point was that the syntax shown above is incorrect because the argument seems to be optional.

Reply:
Yeah, it's fine this way. Maybe `option/flag` could (discretly) hyperlink to that page though.

Reply:
@Zimmi48 is there a canonical way to *name* a warning already?

Reply:
*Somehow my last comment is not shown in the right order (at least to me).*

>is there a canonical way to name a warning already?

Not that I know of.

>Maybe option/flag could (discretly) hyperlink to that page though.

Yes, it would be great if we could have such a mechanism. Feel free to open an issue about that.